test/lib/snCrypto.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  Base64String,
+  HexString,
+  SNPureCrypto,
+  timingSafeEqual,
+  Utf8String,
+} from '@standardnotes/sncrypto-common';
+import CryptoJS from 'crypto-js';
+import sodium from 'libsodium-wrappers';
+import { generateUuid } from '../../lib/utils';
+
+/**
+ * An SNPureCrypto implementation. Required to create a new SNApplication instance.
+ */
+export default class SNCrypto implements SNPureCrypto {
```

Comment:
Wait why do we need crypto operations in this library?

Reply:
You need to provide a valid SNCrypto class otherwise the running SNJS instance will be useless, or at least very far removed from what actual clients are doing. But maybe you can just use sncrypto-web instead

Reply:
True but I guess the question is if we can get away with not providing any implementation in these functions. For example I can't imagine where encryption would be involved with an editor communicating back and forth with SN, as encryption is related to SN client-to-server syncing.

Reply:
> You need to provide a valid SNCrypto implementation otherwise the running SNJS instance will be useless

Yup, this is what I wanted to avoid doing.

> But maybe you can just use sncrypto-web instead

Yes, but since I'm running these tests using a browser-like environment, I would need `WebCrypto` and `WebAssembly`, which is not available in node.

> True but I guess the question is if we can get away with not providing any implementation in these functions.

Seems that you can just return the plaintext string when decrypting/encrypting on these functions, so I'm using that instead.